### PR TITLE
feat(tests): implement testing

### LINKED ISSUE #1: Initialize e2e testing setup
try and see how testing with [TestCafe](https://testcafe.io/) would work. should be discussed (esp. vs cypress)

### PR DESCRIPTION
**Pull request by cfanoulis**
very early, very dirty, barely covers the landing page, but its a start
closes #1

### CODE REVIEW

**Comment by cfanoulis** [comment]
if noone else has any changes to make to ani's changes I'll merge this in 24hrs

File: tsconfig.json [hunk 1]
```diff
@@ -14,6 +14,11 @@
     "isolatedModules": true,
     "jsx": "preserve"
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": [
+    "next-env.d.ts",
+    "**/*.ts",
+    "**/*.tsx",
+    "tests/unit/jest.setup.ts"
```
Comment (jasonappah) [inline tsconfig.json:21]: I think we can remove this since this PR doesn't add unit tests

File: pages/index.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Flex, Heading, Text } from "@chakra-ui/react";
 import Head from "next/head";
+import React from "react";
```
Comment (jasonappah) [inline pages/index.tsx:3]: Just curious, are we going to have to do this on every page?

**Review by jasonappah — CHANGES_REQUESTED** [review]

File: pages/index.tsx (hunk 2 above)
Comment (anirudhb) [reply pages/index.tsx:3]: I don't think this is even a required change

**Review by anirudhb — CHANGES_REQUESTED** [review]
lgtm nit

**Review by jasonappah — APPROVED** [review]

**Pull request merged**